### PR TITLE
Refine API wrapper for "Plugins" once more

### PR DESCRIPTION
**Pull request by amotl**
## About

After GH-110 and GH-114, this patch streamlines the API wrapper about "plugins" a bit more. 

## Details

1. The patch is needed to make https://github.com/panodata/grafana-wtf/pull/91 work.

2. While at it, it also adjusts/trims the method names a bit. I hope it will not cause too much harm, because the feature has only been released recently. I don't think we need to bump to 4.0.0 to properly signal that "breaking" change. I would just run a 3.9.0 release to ship it. Please let me know if you think differently.

/cc @bhks

### CODE REVIEW

File: grafana_client/elements/plugin.py [hunk 1]
```diff
@@ -9,60 +11,83 @@ def __init__(self, client):
         self.client = client
         self.logger = logging.getLogger(__name__)
 
-    def get_installed_plugins(self):
+    def list(self):
         """
-        :return:
+        Return list of all installed plugins.
         """
         path = "/plugins?embedded=0"
```
Comment (bhks) [inline grafana_client/elements/plugin.py:18]: Should we allow user to pass the filters like `embedded=0&core=0` ? Can be a separate PR.

File: grafana_client/elements/plugin.py [hunk 2]
```diff
@@ -9,60 +11,83 @@ def __init__(self, client):
...
-        : return:
+        Install a 3rd-party plugin from the plugin store.
         """
         try:
-            path = "/plugins/%s/install" % pluginId
-            r = self.client.POST(path, json={"version": version})
+            path = "/plugins/%s/install" % plugin_id
+            # Unfortunately, this endpoint may respond with an empty JSON,
+            # which needs compensation, because it does not decode well.
+            r = self.client.POST(path, json={"version": version}, accept_empty_json=True)
```
Comment (bhks) [inline grafana_client/elements/plugin.py:37]: Thanks for handling this , I was not sure on how to handle the exceptions I was getting.

File: grafana_client/elements/plugin.py [hunk 3]
```diff
@@ -9,60 +11,83 @@ def __init__(self, client):
...
             return r
+        except GrafanaClientError as ex:
+            # Ignore `Client Error 409: Plugin already installed`.
+            if "409" not in str(ex):
```
Comment (bhks) [inline grafana_client/elements/plugin.py:41]: Great, I figured after I merged my PR.

**Review by bhks — APPROVED** [review]

File: grafana_client/elements/plugin.py (hunk 1 above)
Comment (amotl) [reply grafana_client/elements/plugin.py:18]: Sure, why not. Can you point us to corresponding documentation or code, where all the possible filtering parameters are enumerated? Maybe there are even more, about sorting or paging?

Comment (bhks) [reply grafana_client/elements/plugin.py:18]: https://github.com/grafana/grafana/blob/main/pkg/api/plugins.go#L48

Comment (amotl) [reply grafana_client/elements/plugin.py:18]: Thanks, see GH-117.
